src/wasm/body.rs

```diff
@@ -52,6 +82,15 @@ impl From<&'static str> for Body {
     }
 }
 
+impl From<Form> for Body {
```

Comment:
I noticed this public `From` differs from the async_impl, we should probably keep it private. It can just be a `pub(super)` method.

Reply:
fixed.